src/runtime_src/core/pcie/driver/linux/xocl/lib/libqdma/qdma_debugfs.h

```diff
@@ -23,7 +23,11 @@
 #include <linux/pci.h>
 #include <linux/debugfs.h>
 #include <linux/fs.h>
+#if defined(__mips__)
+#include <linux/uaccess.h>
+#else
 #include <asm/uaccess.h>
```

Comment:
I think we should just include linux/uaccess.h in all cases. @karenx-xilinx , do you see any problem with including linux/uaccess.h here?

Reply:
Yes, agree. we can just include uaccess.h always.

Reply:
I do not why use asm/uaccess.h, i also think should use linux/uaccess.h in all arches.